spec/unit/search/world_locations_registry_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+RSpec.describe Search::WorldLocationsRegistry do
+  before do
```

Comment:
You could avoid setting these as instance variables using RSpec's [let syntax](https://relishapp.com/rspec/rspec-core/docs/helper-methods/let-and-let) for the slug and [RSpec's implicitly defined subject](https://relishapp.com/rspec/rspec-core/v/3-8/docs/subject/implicitly-defined-subject) for the registry.

```suggestion
  let(:slug) { 'uk-delegation-to-the-ministry-of-magic' }
```